src/algorithms/signal_source/adapters/osmosdr_signal_source.cc

```diff
@@ -93,6 +94,7 @@ OsmosdrSignalSource::OsmosdrSignalSource(const ConfigurationInterface* configura
                     osmosdr_source_->set_gain(gain_, 0);
                     osmosdr_source_->set_if_gain(rf_gain_, 0);
                     osmosdr_source_->set_bb_gain(if_gain_, 0);
+                    osmosdr_source_->set_bandwidth(if_bw_, 0);
```

Comment:
What happens if you set the bandwidth to 0? It's unclear if this is requiring a bandwidth to be set, or truly "override(s) the default". I would imagine that for a GNSS application, a default bandwidth (see line 45) could be asserted.

Reply:
Thanks.
That should not depend on `this->AGC_enabled_` state.
I think, this line is not required as `osmosdr_source` sets optimal bandwidth by default and we should not touch it if we don't have `Signal_source.if_bw` option in a config file.